pkg/virt-controller/watch/vm.go

```diff
@@ -2880,9 +2962,11 @@ func (c *VMController) sync(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachin
 			}
 		}
 
-		err = c.handleCPUChangeRequest(vmCopy, vmi)
-		if err != nil {
-			syncErr = &syncErrorImpl{fmt.Errorf("Error encountered while handling CPU change request: %v", err), HotPlugCPUErrorReason}
+		if lastSeenVMSpec == nil || !reflect.DeepEqual(lastSeenVMSpec.Template.Spec.Domain.CPU, vm.Spec.Template.Spec.Domain.CPU) {
```

Comment:
@jean-edouard better use the `equality` library for DeepEquals. (equality.Semantic.DeepEqual)
from `k8s.io/apimachinery/pkg/api/equality`